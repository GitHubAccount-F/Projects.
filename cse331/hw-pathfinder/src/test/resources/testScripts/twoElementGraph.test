#Tests Dijkstra's algorithm with a graph containing only two
#nodes

#Creates the graph
CreateGraph graph1

#Adds the nodes
AddNode graph1 one
AddNode graph1 two

#Adds edge
AddEdge graph1 one two 10.0
AddEdge graph1 two one 10.0

#Finds a path
FindPath graph1 one two

FindPath graph1 two one