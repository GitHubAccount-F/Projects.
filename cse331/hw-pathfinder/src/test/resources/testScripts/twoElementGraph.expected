#Tests Dijkstra's algorithm with a graph containing only two
#nodes

#Creates the graph
created graph graph1

#Adds the nodes
added node one to graph1
added node two to graph1

#Adds edge
added edge 10.000 from one to two in graph1
added edge 10.000 from two to one in graph1

#Finds a path
path from one to two:
one to two with weight 10.000
total cost: 10.000

path from two to one:
two to one with weight 10.000
total cost: 10.000